mitodl/odl-video-service
# Convert to use Poetry for Python dependency management

#### Pre-Flight checklist



#### What are the relevant tickets?
N/A

#### What's this PR do?
Convert to use Poetry for Python package management

#### How should this be manually tested?
CI tests should pass
You should be able to run the app locally as before, and it should work the same as before:
`docker-compose up`
`docker-compose run --rm web pytest .`
`docker-compose run --rm web python manage.py shell`
etc.

#### Where should the reviewer start?
pyproject.toml

#### Any background context you want to provide?
https://github.com/mitodl/hq/discussions/1422
